salt/cli/daemons.py

```diff
@@ -477,9 +501,11 @@ def start(self):
         try:
             if check_user(self.config['user']):
                 log.info('The proxy minion is starting up')
+                self.verify_hash_type()
+                self.action_log_info('Starting up')
                 self.minion.tune_in()
         except (KeyboardInterrupt, SaltSystemExit) as exc:
-            log.warning('Stopping the Salt Proxy Minion')
+            self.action_log_info('Stopping up')
```

Comment:
@cachedout Removal of direct `log.<level>(<message>)` was because it was always inconsistent and quite messy. So I decided to re-use class names to get pretty much beautiful messages that look always the same. Since those classes are very similar, they are asking to have something in common and have it elsewhere. The same mixin is looking if MD5 is in use in any of those classes.